semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -636,7 +636,8 @@ trait TextDocumentOps { self: SemanticdbOps =>
                 traverse(gtree.check())
 
               case gtree: g.Literal
-                  if gtree.tpe != null && gtree.tpe.typeSymbol.name.decode.contentEquals("Class") =>
+                  if gtree.tpe != null && gtree.tpe.typeSymbol.name.decode
```

Comment:
The idiomatic way to compare symbols is via `==`, you can get a reference to the `java.lang.Class` symbol in `g.definitions.ClassClass`

<img width="559" alt="Screenshot 2019-11-15 at 09 03 40" src="https://user-images.githubusercontent.com/1408093/68961273-db087c80-0786-11ea-92c6-b18f55c7875f.png">

Reply:
Can we add an additional guard that the source code actually contains `classOf`?

Reply:
Fixed.